framework/doc/content/syntax/Controls/index.md

```diff
@@ -91,6 +91,26 @@ In both cases there is an alternative form for defining an object and parameter
 `base::object/name`. In this case "base" is the MOOSE base system that the object is derived from.
 For example, `Kernel::diff/coef`.
 
+## Controllable Parameters Added by Actions
+
+MOOSE also allows parameters in [Action](Action.md) to be controllable.
+The procedure for making a parameter in an [Action](Action.md) controllable is the same as documented in [Creating a Controllable Parameter](syntax/Controls/index.md#sec:control-param).
```

Comment:
You shouldn't need to manually name the page "Creating a Controllable Parameter" - the section name should automatically be the default link text, right?

Reply:
I will have to run the server to double check. I was unsure whether `[syntax/Controls/index.md#sec:control-param]` is enough. I initially had `[sec:control-param]` but for some reason that does not work.

Reply:
`[sec:control-param]` won't be enough, but `[syntax/Controls/index.md#sec:control-param]` should be.